docs/reference/mobile-manipulation.md

```diff
@@ -0,0 +1,269 @@
+# mobile_manipulation module
+
+The *mobile_manipulation* module contains the *MobileRLLearner* class, which inherits from the abstract class *LearnerRL*.
+
+### Class MobileRLLearner
+Bases: `engine.learners.LearnerRL`
+
+The *MobileRLLearner* class is an RL agent that can be used to train wheeled robots for mobile manipulation in conjunction with the
+`mobileRL.env.mobile_manipulation_env` environments and the task wrappers around it in `mobileRL.env.tasks` and `mobileRL.env.tasks_chained`. 
+Originally published in [[1]](#kinematic-feasibility), demonstrations can be found on [Learning Kinematic Feasibility for Mobile Manipulation through Deep Reinforcement Learning](http://kinematic-rl.cs.uni-freiburg.de/).
+
+The [MobileRLLearner](#src.control.mobile_manipulation.mobile_manipulation_learner.py) class has the following public methods:
+
+#### `MobileRLLearner` constructor
+
+Constructor parameters:
+- **lr**: *float, default=1e-5*  
+  Specifies the initial learning rate to be used during training.
+- **iters**: *int, default=1'000'000*  
+  Specifies the number of steps the training should run for.
+- **batch_size**: *int, default=64*  
+  Specifies the batch size during training.
+- **lr_schedule**: *{'', 'linear'}, default='linear'*  
+  Specifies the learning rate scheduler to use. Empty to use a constant rate.
+- **lr_end**: *float, default=1e-6*
+  Specifies the final learning rate if a lr_schedule is used.
+- **backbone**: *{'MlpPolicy'}, default='MlpPolicy'*  
+  Specifies the architecture for the RL agent.
+- **checkpoint_after_iter**: *int, default=20_000*  
+  Specifies per how many training steps a checkpoint should be saved. If it is set to 0 no checkpoints will be saved.
+- **checkpoint_load_iter**: *int, default=0*  
+  Specifies which checkpoint should be loaded. If it is set to 0, no checkpoints will be loaded.
+- **restore_model_path**: *str, default=None*
+  Path to load checkpoints from. Set to 'pretrained' to load one of the provided checkpoints.
+- **temp_path**: *str, default='temp'*  
+  Specifies a path where the algorithm stores log files and saves checkpoints.
+- **device**: *{'cpu', 'cuda'}, default='cuda'*  
+  Specifies the device to be used.
+- **seed**: *int, default=None*  
+  Random seed for the agent. If None a random seed will be used.
+- **buffer_size**: *int, default=100_000*  
+  Size of the replay buffer.
+- **learning_starts**: *int, default=0*  
+  Number of environment steps with a random policy before starting training.
+- **tau**: *float, default=0.001*  
+  Polyak averaging of the target network.
+- **gamma**: *float, default=0.99*  
+  Discount factor.
+- **explore_noise**: *float, default=0.5*  
+  Strength of the exploration noise.
+- **explore_noise_type**: *{'normal', 'OU'}, default=normal*  
+  Type of exploration noise, either normal or Ornstein Uhlenbeck (OU) noise.
+- **ent_coef**: *{'auto', int}, default='auto'*  
```

Comment:
I see your point. Though in this case ent_coef is passed directly on to stable-baselines which is used under the hood. As such it might be helpful to stick with the same argument name they use.